### PR TITLE
TS Lint Errors: Typeahead.tsx and TypeaheadA11yStatus.tsx

### PR DESCRIPTION
**Pull request by csong202**
- deleted min_length props from TypeaheadA11yStatus because it was unused

### CODE REVIEW

File: client/src/components/Typeahead/Typeahead.tsx [hunk 1]
```diff
@@ -54,19 +64,14 @@ interface TypeaheadState {
...
+export class Typeahead extends React.Component<TypeaheadProps, TypeaheadState> {
   menu_id: string;
 
   private typeahead_ref = React.createRef<HTMLDivElement>();
-  private virtualized_list_ref = React.createRef<any>();
+  private virtualized_list_ref = React.createRef<List>();
 
   static defaultProps = {
     placeholder: text_maker("search"),
-    min_length: 3,
```
Comment (Stephen-ONeil) [inline client/src/components/Typeahead/Typeahead.tsx:69]: I think this default's still valid. Let me know if I'm wrong about that!

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]
Thanks for catching the unused `min_length` prop in `TypeaheadA11yStatus`. I think it was moved from there to the screen reader only  `typeahead_usage` text directly in `Typeahead`.

File: client/src/components/Typeahead/Typeahead.tsx (hunk 1 above)
Comment (csong202) [reply client/src/components/Typeahead/Typeahead.tsx:69]: oops didn't mean to delete that one

File: client/src/components/Typeahead/Typeahead.tsx [hunk 2]
```diff
@@ -54,19 +64,14 @@ interface TypeaheadState {
...
-  TypeaheadProps,
-  TypeaheadState,
-  {}
-> {
+export class Typeahead extends React.Component<TypeaheadProps, TypeaheadState> {
   menu_id: string;
 
   private typeahead_ref = React.createRef<HTMLDivElement>();
-  private virtualized_list_ref = React.createRef<any>();
+  private virtualized_list_ref = React.createRef<List>();
```
[the remark this answers is not in the document]
Comment (csong202) [reply client/src/components/Typeahead/Typeahead.tsx:71]: It hasn't caused a problem here since List has recomputeRowHeights

**Review by ktw1016 — APPROVED** [review]
Looks good to me

**Pull request merged**